scripts/linters/pylint_extensions.py

```diff
@@ -590,14 +590,14 @@ class DocstringParameterChecker(checkers.BaseChecker):
     DOCSTRING_SECTION_RAISES = 'raises'
 
     def visit_classdef(self, node):
-        """Visit each class definition in a module and check if there is a
-        single new line below each class docstring.
+        """Visit each class definition and check if there is a
```

Comment:
The docstring should denote what exactly is going to be done here, like what all checks would be done or whatever description seems appropriate. This is not about just one check, right?

Reply:
Oh yes, my bad I forgot to edit it. Thanks!